src/runtime/runtime.ts

```diff
@@ -54,25 +54,313 @@ export class Interner {
     return this.IDs[id];
   }
 
-  release(id: ID) {
+  release(id: ID|undefined) {
+    if(!id) return;
+
     this.IDRefCount[id]--;
     if(!this.IDRefCount[id]) {
       let value = this.IDs[id];
-      this.numbers[value] = undefined;
-      this.strings[value] = undefined;
+      this.numbers[value as number] = undefined;
+      this.strings[value as string] = undefined;
       this.IDFreeList.push(id);
     }
   }
 }
 
 export var GlobalInterner = new Interner();
 
-console.log(GlobalInterner.intern("hey"))
-console.log(GlobalInterner.intern("hey"))
-console.log(GlobalInterner.intern("yo"))
-console.log(GlobalInterner.release(GlobalInterner.get("yo")))
-console.log(GlobalInterner.intern("zomg"))
-console.log(GlobalInterner.intern("yo"))
+/**
+ * A change is a single changed EAV row of a changeset.
+ *  E.g., if we add [#person name: "josh"] then
+ *  (<1>, "tag", "person", ...) and
+ *  (<1>, "name", "josh", ...) are each separate changes.
+ */
+
+class Change {
```

Comment:
We absolutely will once it does something -- right now we're prototyping how the incremental stuff will work, so things that are liable to change are more lightly documented than the more permanent bits. Just for your personal edification though (since you've been kind enough to read along!) `count` represents the multiplicity of the change--that is--how many of the change we're adding or removing.

`Change.fromValues("<1>", "tag", "person", 73, 1, 0 , 3)` would add three of the EAVN `("<1>", "tag", "person", 73)`, while `Change.fromValues("<1>", "tag", "person", 73, 1, 0 , -1)` would remove one of it.

These multiplicities are very important to the incremental model we're trying to adopt, which is based in McSherry's work with Differential Dataflow. It's super cool, and absolutely worth looking into if the subject interests you.